terraform/modules/tf_stream_alert/main.tf

```diff
@@ -61,7 +61,7 @@ resource "aws_s3_bucket" "streamalerts" {
 
 // Legacy S3 bucket name - All alerts should be copied to the bucket created above.
 resource "aws_s3_bucket" "stream_alert_output" {
-  bucket        = "${var.prefix}.${var.cluster}.stream.alert.output.processor.results"
+  bucket        = "${replace("${var.prefix}.${var.cluster}.stream.alert.output.processor.results", "_", ".")}"
```

Comment:
I always have to look twice at the nested double-quotes, but Terraform seems fine with them